.github/workflows/knative-releasability.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is automagically synced here from github.com/knative-sandbox/.github
+# repo by knobots: https://github.com/mattmoor/knobots and will be overwritten.
+
+name: 'Releasability'
+
+on:
+  schedule:
+  - cron: '0 1 * * *' # On the hour.
+
+jobs:
+  releasability:
+    name: Releasability
+    runs-on: 'ubuntu-latest'
+
+    env:
+  #########################################
+  #   Update this section each release.   #
+      RELEASE: v0.19
+      SLACK_CHANNEL: '@n3wscott'
+  #########################################
+      SLACK_ICON: http://github.com/knative.png?size=48
+      SLACK_USERNAME: github-actions
+      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+      MSG_MINIMAL: 'true'
+      SLACK_TITLE: Releasability for ${{ github.repository }} is UNKNOWN
+
+    steps:
+      - name: Set up Go 1.15.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        run: |
+          GO111MODULE=on go get tableflip.dev/buoy@master
```

Comment:
This needs to happen outside of the module or it will modify go.mod

Reply:
that might matter later, ok will fix